client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -114,7 +115,7 @@ const VisitDashboardItem = ( { site }: { site: SiteExcerptData } ) => {
 	);
 };
 
-export default function SitesTableRow( { site }: SiteTableRowProps ) {
+export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
```

Comment:
@p-jackson Out of curiosity, I thought the component won't re-render unless its props change? What am I missing, and what does `memo` give us in this case?

Reply:
@danielbachhuber that is not correct for nested components.
Here is an example to prove it: https://codesandbox.io/s/memo-is-your-friend-fhtp7x?file=/src/App.js
If the parent changes, the child will execute the render unless it is memoized.

Reply:
`memo` skips rendering if the props are equal. If not using `memo`, it will render if the parent changes regardless if the props are the same or not.